src/components/Chat/Chat.tsx

```diff
@@ -1563,10 +1733,48 @@ export const Chat = memo(
 
         try {
           // Update localStorage
-          localStorage.setItem(
-            'selectedConversation',
-            JSON.stringify(updatedConversation),
-          )
+          try {
+            localStorage.setItem(
+              'selectedConversation',
+              JSON.stringify(updatedConversation),
+            )
+          } catch (storageError) {
+            // Handle localStorage quota exceeded error
```

Comment:
Nice error handling here, we get hit with `quota exceeded error` from time to time.